source/Nevermore.IntegrationTests/Advanced/HooksFixture.cs

```diff
@@ -22,13 +22,15 @@ public void ShouldCallHooks()
             var customer = new Customer {Id = "C131", FirstName = "Fred", LastName = "Freddy"};
             transaction.Insert(customer);
             AssertLogged(log, "BeforeInsert", "AfterInsert");
-            
+
+            customer = transaction.Load<Customer>("C131");
```

Comment:
So once I insert, I need to reload to make an update? Could Nevermore fill in the `RowVersion` for me?

Reply:
👍